optimizely/project_config.py

```diff
@@ -514,6 +515,15 @@ def get_anonymize_ip_value(self):
 
         return self.anonymize_ip
 
+    def get_send_flag_decisions_value(self):
+        """ Gets the Send Flag Decisions value.
+
+        Returns:
+            A boolean value that indicates if we should send flag decisions.
+        """
+
+        return self.send_flag_decisions
+
```

Comment:
Can you add some tests in this file like we have for bot filtering: https://github.com/optimizely/python-sdk/blob/master/tests/test_config.py#L602-L614